Makefile

```
@@ -8,7 +8,12 @@ NOLINT=0
 
 SRCS_DIR:=src
 LIBS_DIR:=lib
-OBJS_DIR:=bin
+OBJS_DIR:=bin/
```

This seems like a redundant `/`
```suggestion
OBJS_DIR:=bin
```